Makefile

```diff
@@ -50,3 +56,11 @@ docker-serve:
 test-examples:
 	scripts/test_examples.sh install
 	scripts/test_examples.sh run
+
+link-checker-setup:
```

Comment:
This target action is:
get/pull htmltest docker image?

Reply:
@kbhawkey yes.